src/backend/distributed/commands/role.c

```diff
@@ -198,6 +273,74 @@ WrapQueryInAlterRoleIfExistsCall(const char *query, RoleSpec *role)
 }
 
 
+static const char *
+CreateQueryArray(List *stmts)
```

Comment:
missing function comment

Reply:
function became obsolete due to propagating grants separately